KaiDevrim.md

```diff
@@ -0,0 +1,47 @@
+---
+name: "@KaiDevrim"
+project: "NFC Unlock & Stream Deck & Soham Microphone"
+---
+
+# Project Name
+NFC Unlock & Stream Deck & Soham Microphone
+
+## Summary
+I want to make a system where depending on who wants to use the computer, the pi will detect them via their keycard and log them into all the websites under their account and everything. This will be very useful if I want to use this as a guest computer or was teaching someone and they needed to use my computer.
+
+For the Stream Deck, I just want basically a hotkey keyboard to do advanced actions, like custom scripts and whatnot.
+
+I finally want to make a simple monitor backlight that basically just detects what is on screen and changes the light of an led strip to match that colour. It will be very useful for immersive games as well as immersive movies.
+
+## Plan
+
+### NFC
+I plan on using a raspberry pi that I already own, along with a pikvm so it acts as a troubleshooting device for my pi. My raspberry pi will use the PiKVM to act as a keyboard and mouse, and will also allow me to remotely monitor the user in case they need help. I will be using a an nfc card with the nfc pi hat to do all the authentication and stuff and everything. 
+
+### Stream Deck
+For the Stream Deck, I will use a mechanical keypad, attach that to my raspberry pi, and use the OSS https://stream-pi.com/ to control my computer, phone, server, lights, all from a click of a button. I will need the SD card switcher so that I can use one pi for multiple projects.
+
+### Soham Microphone
+For the micrphone, Soham plans on justing follow this very simple video, [Building a quality USB-C microphone](https://www.youtube.com/watch?v=LoQu3XXIayc). He also plans on chipping in the rest of his budget to this. 
```

Comment:
Unfortunately, grant funds aren't able to be transferred across different projects. The only case where a project can receive a grant of $250+ is if there are multiple people working on the same project. It looks like Soham is planning to work on a project of his own.

Reply:
With that said, would you still be interested in pursuing the proposed additions to your project?

Reply:
If so, please request me for another review, and I'll take a look as soon as I can.

Reply:
Hi, so Kai and I will be working on this together! It's a joint project, but the total price is staying under 250? We're both building our own projects but we're also pooling our extra funds to work on this one together.